conf.yaml

```diff
@@ -34,6 +34,7 @@ repos:
     logstash:             https://github.com/elastic/logstash.git
     logstash-docs:        https://github.com/elastic/logstash-docs.git
     sense:                https://github.com/elastic/sense.git
+    endpoint-app-team:    https://github.com/elastic/endpoint-app-team
```

Comment:
Should this be: `https://github.com/elastic/endpoint-app-team.git` (I don't know)?